apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -15,7 +15,7 @@
     <!-- region ledger table-->
     <div class="ledg-head transactions-page mr-t15 position-relative">
         <!-- region ledger header -->
-        <div class="clearfix row mr-b15">
+        <div class="clearfix row mr-b15" [ngClass]="{'mr-b5' :  currentCompanyBranches?.length > 2}">
```

Comment:
Please check for scenario with company branches > 2 and user is switched to branch

Reply:
this mr-b5 class will be applied only when a company has a branch of more than 2 , working correctly now